modules/validate-json-schema/test/index.js

```diff
@@ -0,0 +1,160 @@
+const validateJSONSchema = require('..');
+const expect = require('chai').expect;
+
+describe('ValidateJSONSchema', function () {
+  var request = {
+    headers: {
+      accept: 'application/vnd.api+json'
+    },
+    query: {},
+    payload: {},
+    body: {},
+    auth: {}
+  };
+
+
+  it('should return null when there is no schema on the endpoint', function() {
```

Comment:
We don't need to test all the different forms here since we now allow testing any keys.